city_scrapers/spiders/il_port_district.py

```diff
@@ -0,0 +1,181 @@
+import datetime
+import re
+
+import scrapy
+from city_scrapers_core.constants import BOARD, COMMITTEE, NOT_CLASSIFIED
+from city_scrapers_core.items import Meeting
+from city_scrapers_core.spiders import CityScrapersSpider
+
+
+class IlPortDistrictSpider(CityScrapersSpider):
+    name = "il_port_district"
+    agency = "Illinois International Port District"
+    timezone = "America/Chicago"
+    allowed_domains = ["https://www.iipd.com"]
+    start_urls = [
+        "https://www.iipd.com/calendar/agendas", "https://www.iipd.com/about/board-meeting-minutes",
+        "https://www.iipd.com/calendar/schedules"
+    ]
+    location = {
+        "name": "Illinois International Port District ",
+        "address": "3600 E. 95th St. Chicago, IL 60617",
+    }
+
+    def start_requests(self):
+        start_urls = [
```

Comment:
Since you're doing something different in each section and the sections each depend on the results of the previous page you might need to only parse one of the document pages and then yield a request to the next one so that you can guarantee the schedules are parsed last. You could also implement the `spider_idle` signal to yield a request to the schedule page like in [`il_pollution_control`](https://github.com/City-Bureau/city-scrapers/blob/a11b7c4dda9018efcd3e3f25b7678546a1351cbe/city_scrapers/spiders/il_pollution_control.py)